platform/engineering/design-docs/2020-12-10-auto-deploy-cms-build.md

```diff
@@ -71,21 +57,21 @@ To deploy content changes at will, that "auto-deploy" job is manually triggered
 
 - Content writers want to quickly draft, publish, and deploy content.
   - However, `vets-website` is only deployed to production once per day.
-  - The CMS team can run content-only deploys independently of the daily production deploy, but that is triggered manually.
+  - The CMS team can run content-only deploys independently of the daily production deploy, but that must be triggered manually.
 - The content-only deploy does not include automated accessibility checks.
 - In local development, engineers can encounter confusing build errors that are due to the mismatch in outdated locally cached content and updated Liquid templates.
 
 ### High Level Design
 
 There will be a new content-only auto-deploy job in addition to the existing full auto-deploy.
-- It will build content with CMS export data.
+- It will build content with GraphQL query data.
 - It will be a recurring job that executes on a more frequent schedule than the daily prod deploy.
 
 The initial schedule will run between 8am and 8pm ET to align with CMS support hours.
```

Comment:
During testing, I would run the script 9AM - 4PM so that the team can jump on it if there are any issues. Maybe even cut it off at 3PM. We can still run it every X hours it's just that the team will be around.

Reply:
This also brings up a good point in that what time do we want to run the auto deploy since we are starting out only running it once per day for the initial release? cc @timwright12.

Reply:
@theodur  I vote for a morning-ish deploy since we have the 3pm ET one going out and it fails a lot right now. If we do something around 11am ET that should give us enough time to have a clean full deploy if something goes wrong.

Reply:
@timwright12 Yup I was thinking the same thing. I went with a morning time in my draft [PR](https://github.com/department-of-veterans-affairs/devops/pull/8568) yesterday for updating the cron in the trigger. I added 10 AM but since you mentioned 11 AM we can make it that. I was debating between those two hours specifically.